source/_posts/face-detection-with-dlib.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Face Detection with dlib
+date: 2018-10-18
+author: Isaac Zepeda
+author_id: izepeda
+tags:
+  - face-detection
+  - machine-learning
+  - computer-vision
+  - dlib
+---
+
+<img src="/face-detection-with-dlib/cover.jpg" />
+
+Face Detection is the first step towards Face Recognition.
+
+There are a number of algorithms to detect faces: [haar cascade](https://www.quora.com/What-is-haar-cascade), [Histogram of Oriented Gradients](https://en.wikipedia.org/wiki/Histogram_of_oriented_gradients), [Neural Networks](https://en.wikipedia.org/wiki/Artificial_neural_network). There is a [Haar implementation in OpenCV](https://docs.opencv.org/3.4.2/d7/d8b/tutorial_py_face_detection.html), and dlib uses HOG and [Neural Networks](https://towardsdatascience.com/cnn-based-face-detector-from-dlib-c3696195e01c).
+
+We'll learn how to detect faces in an image in few lines of code using python, [dlib](http://dlib.net/) and [opencv](https://opencv.org/).
+
+<!-- more -->
+
+## The Libraries
+
+[Dlib](http://dlib.net/) is a modern C++ toolkit containing machine learning algorithms and tools for creating complex software to solve real world problems. We are gonna use its [Python API](http://dlib.net/python/index.html).
+
+[OpenCV](https://opencv.org/) is a Computer Vision library also with interfaces in C++, Python and Java, and contains a lot of functions to help us to manipulate images.
+
+
+## Installing Dlib and Opencv
+
+Installing Python and its libraries is out of scope of this post. There are some tutorials out there that would help you get python, dlib and opencv installed. Here are a couple of links to install them on mac os:
```

Comment:
`Installing Python and its libraries **are out of the scope** of this post. There are some tutorials out there that would help you get python, dlib, and **OpenCV** installed. Here are a couple of links to install them on mac os`

Reply:
Why "OpenCV" as bold?

Reply:
Ahh nevermind, I see it's a correction not a markdown format change request :P